cln_lspd.go

```diff
@@ -188,19 +189,25 @@ func OnHtlcAccepted(event *glightning.HtlcAcceptedEvent) (*glightning.HtlcAccept
 		if fundingTxID == nil {
 			if hex.EncodeToString(paymentHash) == event.Htlc.PaymentHash {
 
-				fundingTxID, fundingTxOutnum, err = clnOpenChannel(clientcln, hex.EncodeToString(paymentHash), hex.EncodeToString(destination), incomingAmountMsat)
+				fundingTxID, fundingTxOutnum, err := clnOpenChannel(clientcln, hex.EncodeToString(paymentHash), hex.EncodeToString(destination), incomingAmountMsat)
```

Comment:
This will initialize a new variable fundingTxID  which shadows the existing one. Later on in this function the code expects an initialized fundingTxID